v3.0/how-to/migrate/from-aurora.md

```diff
@@ -42,11 +42,11 @@ DM 在增量同步阶段依赖 `ROW` 格式的 binlog，如果未启用 binlog 
 
```

Comment:
Please also update the link in line 21 and then apply the same changes to `dev` and `v2.1`.

Reply:
@lilin90 Fixed in https://github.com/pingcap/docs-cn/pull/1569

Reply:
OK.